ksp_plugin/interface.cpp

```diff
@@ -360,6 +360,7 @@ double principia__current_time(Plugin const* const plugin) {
 }
 
 char const* principia__SerializePlugin(Plugin const* const plugin) {
+  LOG(INFO) << __FUNCTION__;
```

Comment:
It's aperiodic (called only on scene load/save) which should be `INFO` per #347.